lib/timezone_teleporter.rb

```diff
@@ -17,12 +17,28 @@ def configuration
       @configuration ||= Configuration.new
     end
 
-    def teleport(lat, lng)
-      TIMEZONE_LOCATIONS[timezone_at(lat, lng)]
+    def teleport(input)
```

Comment:
an "idiomatic" way to do this in ruby would be:
```ruby
def teleport(lat_or_tz, long=nil)
  if long.nil?
    teleport_timezone(lat_or_tz)
  else
    teleport_location(lat_or_tz, long)
  end
end
```

Reply:
🤯🔥